make-release.sh

```diff
@@ -62,9 +62,14 @@ if [[ "${BASEBRANCH}" != "${BRANCH}" ]]; then
 fi
 
 apply_files_edits () {
-  THEIA_VERSION=$(curl --silent http://registry.npmjs.org/-/package/@theia/core/dist-tags | sed 's/.*"next":"\(.*\)".*/\1/')
+  BUILD_INCLUDE_THEIA_COMMIT_SHA=$(grep -e "^THEIA_COMMIT_SHA=" build.include | cut -d '=' -f2 | tr -d '"')
+  if [ -z "${BUILD_INCLUDE_THEIA_COMMIT_SHA}"]; then
+    THEIA_VERSION=${BUILD_INCLUDE_THEIA_COMMIT_SHA}
```

Comment:
Here it won't work

THEIA_VERSION format is for example
1.16.0-next.3716c071
1.15.0

and not the sha1

so we need to take first digits of BUILD_INCLUDE_THEIA_COMMIT_SHA and then search a package matching this sha1


for example let say commit sha1 is `3716c0716f4f7fe30ef5b36faff4f85e833b20be` we need to take `3716c071`
```bash 
$ npm view @theia/core versions | grep 3716c071                                                                                                                                                                               
'1.16.0-next.3716c071',
```

then we can take `1.16.0-next.3716c071` for THEIA_VERSION

Reply:
curl command to get all versions
```bash
$ curl -H "Accept: application/vnd.npm.install-v1+json" https://registry.npmjs.org/@theia/core | jq -r '.versions | keys[]'
...

1.7.0-next.ff9b62f9
1.8.0
1.8.0-next.16fec095
1.8.0-next.1e603511
1.8.0-next.33886b52
1.8.0-next.56c22749
1.8.0-next.5c164f7f
1.8.0-next.61200efc
1.8.0-next.632a67a5
1.8.0-next.cd85908d
1.8.0-next.e2098321
1.8.0-next.fd0a8ce0
1.8.0-next.fea40a2c
1.8.1
...
```